mysite/profile/tests.py

```diff
@@ -1891,7 +1879,7 @@ def add_two_people_to_a_project_and_send_emails(
                 email_contexts[participant.pk] = email_context
             output = email_contexts
 
-        return (participants_who_are_news_to_each_other, output)
+        return participants_who_are_news_to_each_other, output
```

Comment:
Technically, tuples do not require the parens. However for clarity, I have no problem putting them back in.